scripts/zones/Waughroon_Shrine/mobs/Metsanneitsyt.lua

```diff
@@ -11,6 +11,12 @@ entity.onMobInitialize = function(mob)
     mob:setMod(xi.mod.SLEEPRES, 50)
 end
 
+entity.onMobEngaged = function(mob, target)
+    mob:timer(3000, function(mobArg)
+        mobArg:useMobAbility(686)
+    end)
```

Comment:
Nice idea. I'll have to steal this to prevent sky gods from casting the moment they spawn.